workflow/hosts/hera.yaml

```diff
@@ -1,4 +1,5 @@
-BASE_GIT: '/scratch1/NCEPDEV/global/glopara/git'
+# BASE_GIT: '/scratch1/NCEPDEV/global/glopara/git'
+BASE_GIT: '/scratch2/NCEPDEV/ensemble/save/Walter.Kolczynski/gw_external_git'
```

Comment:
Is this temporary?

Reply:
Very. Will change it back as soon as Kate gets the modified versions installed in glopara. I never even updated the WCOSS one, but it has a similar path.

Reply:
New versions have now been installed in the glopara spaces, so these have been reverted.